README.md

```diff
@@ -22,113 +22,113 @@ These numbers are derived from local simulations with fixed inputs (see [Run ben
 <details>
 <summary><b>Run options</b></summary>
 
-Last run: Wed, 21 Feb 2024 05:53:32 GMT
-| Option              |   Value |
-| :------------------ | ------: |
-| L2 gas price (Gwei) | `0.004` |
-| L1 gas price (Gwei) |    `40` |
-| ETH price (USD)     | `$2500` |
+Last run: Thu, 25 Apr 2024 17:35:36 GMT
+| Option                |         Value |
+| :-------------------- | ------------: |
+| L2 gas price (Gwei)   |        `0.06` |
+| L1 fee per gas (Gwei) | `0.887435714` |
```

Comment:
Actually we can probably revert this. Fee per gas is gas price. Just wanted to make clear it's now not the L1 gas price but the DA cost

Reply:
made a take 2 of this